server/sonar-server/src/main/java/org/sonar/server/issue/index/IssueDoc.java

```diff
@@ -144,10 +144,14 @@ public String resolution() {
     return getNullableField(IssueIndexDefinition.FIELD_ISSUE_RESOLUTION);
   }
 
+  /**
+   * @deprecated since 5.5
+   */
+  @Deprecated
   @Override
   @CheckForNull
   public String reporter() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)